ppr-ui/src/utils/ppr-api-helper.ts

```diff
@@ -28,13 +29,20 @@ import { SearchHistoryResponseIF } from '@/interfaces/ppr-api-interfaces/search-
  * 500 INTERNAL_SERVER_ERROR
  */
 
-export const successfulPPRResponses = [StatusCodes.OK, StatusCodes.CREATED, StatusCodes.ACCEPTED]
+export const successfulPPRResponses = [
```

Comment:
why all the style changes in this file :( ? If it's passing the linter and you're not changing the logic then I think it's a lot nicer to leave this out of the diff. If you want to change things like this put it in a separate PR please

Reply:
I just probably ran the linter on it................................

Reply:
darn haha